mcs/class/System.Data/Test/ProviderTests/System.Data.SqlClient/SqlConnectionTest.cs

```diff
@@ -287,12 +287,22 @@ public void Close ()
 		[Test]
 		public void ChangeDatabase ()
 		{
-			if (ConnectionManager.Instance.Sql.IsAzure)
-				Assert.Ignore("SQL Azure doesn't support 'ChangeDatabase'");
+			conn = new SqlConnection(connectionString);
+			conn.Open();
 
-			conn = new SqlConnection (connectionString);
-			conn.Open ();
-			conn.ChangeDatabase ("master");
+			bool isAzure = ConnectionManager.Instance.Sql.IsAzure;
+
+			try
+			{
+				conn.ChangeDatabase("master");
+			}
+			catch (SqlException exc) when (isAzure)
+			{
+				Assert.AreEqual(40508, exc.Number); //USE statement is not supported to switch between databases (Azure).
+				return;
+			}
+
+			Assert.IsFalse(isAzure);
```

Comment:
@akoeplinger I actually tried to keep the same code-style the whole ProviderTests was written with. So it might be a good idea to re-format everything to actual code convention.